packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1994,6 +1994,9 @@ export default class Meeting extends StatelessWebexPlugin {
 
     return MeetingUtil.validateOptions(options)
       .then(() => {
+        if (!this.mediaProperties.peerConnection) {
+          this.mediaProperties.setMediaPeerConnection(MediaUtil.createPeerConnection());
```

Comment:
Is this an async operation? Should it be added to the promise chain?

Reply:
its not a async operation so it should be fine for now